resources/queries/targetedms/PTMPercentsGrouped.sql

```diff
@@ -0,0 +1,29 @@
+SELECT
+    PeptideGroupId,
+    SiteLocation,
+    PeptideModifiedSequence,
+    Sequence @hidden,
+    -- We have a special rule for this C-term modification - we're actually interested in the _unmodified_ percentage
+    (CASE WHEN Modification.Name = 'Lys-loss (Protein C-term K)' THEN 'C-term K' ELSE Modification.Name END) AS Modification,
+    MIN(Id) AS Id @hidden,
+    PreviousAA @hidden,
+    NextAA @hidden,
+    SampleName,
+    (CASE WHEN Modification.Name = 'Lys-loss (Protein C-term K)' THEN (1 - MAX(MinPercentModified)) ELSE MAX(MaxPercentModified) END) AS MaxPercentModified,
```

Comment:
Is 1 - MAX(MinPercentModified) correct? or should it be 1 - MAX(MaxPercentModified)

Reply:
Yes, it's correct. I added a comment to help clarify. Also, there will only be a single value for MinPercentModified and MaxPercentModified, so it doesn't matter if we do a MIN or a MAX to get it through the aggregate. I flipped this part of the expression to MIN(MinPercentModified) because it reads better and gives identical results.